File: RGLockboxExample/RGLockboxExample/ViewController.m
//
//  ViewController.m
//  RGLockboxExample
//
//  Created by Ryan Dignard on 9/5/16.
//  Copyright © 2016 Ryan Dignard. All rights reserved.
//

@import RGLockboxIOS;

#import "ViewController.h"

@interface ViewController ()

@property (nonatomic, weak) IBOutlet UITextField* keyField;
@property (nonatomic, weak) IBOutlet UITextField* valueField;

@end

@implementation ViewController

- (IBAction)pressedRead {
//    RGLockbox* manager = [RGLockbox manager];
//    self.valueField.text = [[NSString alloc] initWithData:[manager dataForKey:self.keyField.text] encoding:NSUTF8StringEncoding];
}

- (IBAction)pressedSet {
//    RGLockbox* manager = [RGLockbox manager];
//    [manager setData:[self.valueField.text dataUsingEncoding:NSUTF8StringEncoding] forKey:self.keyField.text];
}

@end

File: RGLockboxExample/RGLockboxExample/ViewController.swift
//
//  ViewController.swift
//  RGLockboxExample
//
//  Created by Ryan Dignard on 8/7/16.
//  Copyright © 2016 Ryan Dignard. All rights reserved.
//

import UIKit
import RGLockboxIOS

let theKey = "theKey"

class ViewController: UIViewController {

    @IBOutlet weak var textField: UITextField?
    @IBOutlet weak var saveButton: UIButton?
    @IBOutlet weak var recallButton: UIButton?
    
    override func viewDidLoad() {
        super.viewDidLoad()
        self.textField?.text = RGLockbox.manager().stringForKey(theKey)
    }

    @IBAction func pressedSave() {
        RGLockbox.manager().setString(self.textField?.text, key: theKey)
    }
    
    @IBAction func pressedRecall() {
        self.textField?.text = RGLockbox.manager().stringForKey(theKey)
    }
}

